Open Judge System/Data/OJS.Data.Models/Contest.cs

```diff
@@ -40,6 +40,9 @@ public Contest()
         [Index]
         public bool IsVisible { get; set; }
 
+        [Required]
+        public bool AutoChangeTests { get; set; }
```

Comment:
If you don't know the system, what information will give you this name? Maybe the name must be more concrete